lua/lualine/components/aerial.lua

```diff
@@ -74,12 +82,13 @@ local function format_status(symbols, depth, separator, icons_enabled, colored)
   for _, symbol in ipairs(symbols) do
     if icons_enabled then
       local icon = color_icon(symbol.kind, symbol.icon, colored)
-      table.insert(parts, string.format("%s %s", icon, symbol.name))
+      local name = color(symbol.kind, symbol.name, colored)
```

Comment:
You could extract this to outside of the if-block so you don't have to repeat it on line 88

Reply:
I think this can be resolved, implemented in next commit.